packages/compass/src/index.d.ts

```diff
@@ -45,6 +45,8 @@ declare module 'process' {
 
         COMPASS_CSFLE_SUPPORT?: 'true' | 'false';
 
+        COMPASS_CLUSTERED_COLLECTIONS?: 'true' | 'false';
```

Comment:
Feature-flag we will have to remember to remove at/after mongodb world!!!

Reply:
Do we have a ticket to remove it? Might help keep track of it so we can't easily forget.